src/comparator.js

```diff
@@ -69,17 +71,85 @@ function normalise (entry: any) {
     'Customer Address Line 2': null
   }
 }
+
+let alternativeCountryNames = new Map([
+  ['GBR', 'UNITED KINGDOM'],
+  ['AU', 'AUSTRALIA'],
+  ['USA', 'UNITED STATES'],
+  ['CA', 'CANADA'],
+  ['THE NETHERLANDS', 'NETHERLANDS'],
+  ['HOLLAND', 'NETHERLANDS'],
+  ['TANZANIA', 'TANZANIA, UNITED REPUBLIC OF'],
+  ['RUSSIA', 'RUSSIAN FEDERATION'],
+  ['ANTIGUA', 'ANTIGUA AND BARBUDA'],
+  ['MACAU', 'MACAO'],
+  ['SOUTH KOREA', 'KOREA, REPUBLIC OF'],
+  ['IRAN', 'IRAN, ISLAMIC REPUBLIC OF'],
+  ['LAOS', 'LAOS, PEOPLE\'S DEMOCRATIC REPUBLIC OF'],
+  ['BRITISH VIRGIN ISLANDS', 'VIRGIN ISLANDS, BRITISH'],
+  ['KYRGYSTAN', 'KYRGYZSTAN'],
+  ['BOSNIA-HERZEGOVINA', 'BOSNIA AND HERZEGOVINA'],
+  ['CAPE VERDE ISLANDS', 'CAPE VERDE'],
+  ['FALKLAND ISLANDS', 'FALKLAND ISLANDS (MALVINAS)'],
+  ['VIETNAM', 'VIET NAM'],
+  ['TRINIDAD AND TOBAGO', 'TRINIDAD & TOBAGO']
+])
+
+function normaliseWeekly (entry: any) {
+  let address1 = entry['Address 1'] || ''
+  let normalisedName = entry['Name']
+    .replace('Mr. ', 'Mr ')
+    .replace('Dr. ', 'Dr ')
+    .replace('Prof. ', 'Prof ')
+    .replace('Mrs. ', 'Mrs ')
+    .replace('Ms. ', 'Ms ')
+    .replace('Mx. ', 'Mx ')
+    .replace('Rev. ', 'Rev ')
+    .replace('MR. ', 'MR ')
```

Comment:
Shouldn't the right-hand-side of these be lower case for the next 7 lines?

Reply:
No, we do this is because the files for Australia, NZ and a couple more  capitalize the name including the title.
So in those files we want to replace 'MR. MARTY MCFLY' with 'MR MARTY MCFLY'.